src/middleware/error-handler.js

```diff
@@ -1,16 +1,21 @@
-const logger = require('../utils/logger');
+const { logger } = require("../utils/logger");
 
-const errorHandler = (
-  err,
-  req,
-  res,
-  next
-) => {
-  logger.error(err.message);
-  if (!err.statusCode) err.statusCode = 500;
-  res.status(err.statusCode).send(err.message);
+const ERROR_CODES = {
+  UNPROCESSABLE_ENTITY: 422,
+  NOT_FOUND: 404,
+  INTERNAL_SERVER_ERROR: 500
+};
+
+const errorHandler = (err, req, res, next) => {
+  console.log("raw", err);
+  const error = JSON.parse(err.message);
+  console.log("error", error);
+  logger.error(error.status);
```

Comment:
Why are there two different methods of logging here? It's better to pick one and be consistent.

It looks like you're just logging the error status code here. That doesn't give a whole lot of information, because you'd just see a number appear in the log. I would do something like

```
logger.error("Received `error.status` error")
```

just to add a bit of context

Reply:
Because that `console.log` is a dumb leftover from testing. 😄 
It's removed!

Reply:
I don't know that the message makes that much of a difference, because winston (in the `logger` util) help identify the error already.
The actual log shows
```
error:  404
info:   GET /community_groups/1000 404 18 - 7.440 ms
```


I did change the `error-handler` to log this though:
```
 logger.error(`${error.status} - ${error.message}`);
``` 
This should help with cases where the actual message provides additional information